app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from .elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+        self, nickname: str, musical_instrument: str, bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (
+            f"Elf ranger {self.nickname}. "
+            f"{self.nickname} has bow of the {self._bow_level} level"
+        )
+
+    def get_rating(self) -> int:
```

Comment:
The `get_rating` method should have a return type annotation of `int` to match its implementation. Please update the method signature to reflect this.